docs/source/libraries.rst

```diff
@@ -7,49 +7,53 @@ Here is where the Libraries API's descriptions are located:
-| Motor PWM   | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| TWAI        | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| Ethernet    | Yes         | Yes     | Only for ESP32     |
-+-------------+-------------+---------+--------------------+
-| USB         | Yes         | Yes     | Only for ESP32-S2  |
-+-------------+-------------+---------+--------------------+
++---------------+---------------+---------------+---------------+--------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | Comments           |
++===============+===============+===============+===============+====================+
+| ADC           | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Bluetooth     | Yes           | Not Supported | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| DAC           | No            | No            | No            | Planned            |
++---------------+---------------+---------------+---------------+--------------------+
+| Ethernet      | Yes           | Not Supported | Not Supported |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| GPIO          | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Hall Sensor   | Yes           | Not Supported | Not Supported |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| I2C           | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| I2S           | No            | No            | No            | WIP                |
++---------------+---------------+---------------+---------------+--------------------+
+| LEDC          | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Motor PWM     | No            | Not Supported | Not Supported |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Pulse Counter | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| RMT           | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| SDIO/SPI      | Yes           | Yes           | Yes           |                    |
```

Comment:
SDIO and SPI are not the same thing. SDIO AFAIK is supported only on ESP32 and does not have Arduino API